embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -44,11 +44,38 @@ while true; do
     esac
 done
 
+embulk_args="$@"
+
+while [ $# -gt 0 ] ; do
+    case "$1" in
+        "-b" | "--bundle")
+            shift
+            export EMBULK_BUNDLE_PATH="$1"
+            shift
+            break
+            ;;
+        *)
+            shift
+            ;;
+    esac
+done
+
+if test -z ${EMBULK_BUNDLE_PATH}; then
+    unset EMBULK_BUNDLE_PATH
+    unset BUNDLE_GEMFILE
+    export GEM_HOME="$(cd && pwd)/.embulk/jruby/$(java -cp $0 org.jruby.Main -e 'print RbConfig::CONFIG["ruby_version"]')"
```

Comment:
Oh. I didn't know `$()` POSIX-compliant. 
I tested some commands on FreeBSD 10.
It seems that `$()` and `unset` work well.

I'm not familiar bash on Windows too. 

```bash
$ A=`pwd`
$ echo $A
/home/vagrant
$ B=$( pwd )
$ echo $B
/home/vagrant
$ unset B
$ echo $B

$
```